extension/src/lib.rs

```diff
@@ -13,12 +13,15 @@ lazy_static! {
         .unwrap_or_else(|_| String::from("https://boderator.armaforces.com/api"));
     static ref ATTENDANCE_API: String = std::env::var("AF_ATTENDANCE_API")
         .unwrap_or_else(|_| String::from("https://armaforces.com/api"));
+    static ref SERVER_API: String = std::env::var("AF_SERVER_API")
+        .unwrap_or_else(|_| String::from("https://server.armaforces.com:8888"));
```

Comment:
TODO we could hide this behind some sort of short living cache to not strain the game server with requests